content/notes/create-wp-redis-instance.md

```diff
@@ -12,8 +12,12 @@ emoji: 📰
 Here's a step-by-step copy-paste tutorial:
 
 ```bash
+# Setup permissions in case you run PHP-FPM under apache user. If not this step can be skipped.
+chown admin12:apache /home/virtual/benchmark.test/var/www/redis.sock
+chmod 0660 /home/virtual/benchmark.test/var/www/redis.sock
+
 # Create a new Redis instance for benchmark.test named "wp-test" listening on /tmp/redis.sock
-cpcmd -d benchmark.test redis:create wp-test '[unixsocket:/var/tmp/redis.sock]'
+cpcmd -d benchmark.test redis:create wp-test '[unixsocket:/var/www/redis.sock]'
```

Comment:
Rewrite to `cpcmd -d benchmark.test redis:create wp-test '[unixsocket:/var/www/redis.sock,unixsocketperm:0660]'`

This would set the permissions correctly on each startup.